source/documentation/3.9/rspec-expectations/method_list.html

```diff
@@ -1558,72 +1558,72 @@ <h1 id="full_list_header">Method List</h1>
 
   <li class="even ">
     <div class="item">
-      <span class='object_link'><a href="RSpec/Matchers.html#include-instance_method" title="RSpec::Matchers#include (method)">#include</a></span>
-      <small>RSpec::Matchers</small>
+      <span class='object_link'><a href="RSpec/Matchers/BuiltIn/RespondTo.html#ignoring_method_signature_failure!-instance_method" title="RSpec::Matchers::BuiltIn::RespondTo#ignoring_method_signature_failure! (method)">#ignoring_method_signature_failure!</a></span>
+      <small>RSpec::Matchers::BuiltIn::RespondTo</small>
```

Comment:
These changes don't look like spacing changes... :S

Reply:
There is some reordering. I don't no why but it doesn't seem to have impact in method list.

<img width="533" alt="Capture d’écran 2020-04-08 à 12 32 13" src="https://user-images.githubusercontent.com/8417720/78774465-ff166580-7994-11ea-8020-b995931f98a5.png">